bd_spot_wrapper/spot_wrapper/april_tag_pose_estimator.py

```diff
@@ -0,0 +1,101 @@
+from typing import List, Tuple
+
+import cv2.aruco as aruco
+import fairotag as frt
+import magnum as mn
+import sophus as sp
+
+# MAYBE: we dont need spot
+from spot_wrapper.spot import Spot
+
+MARKER_LENGTH = 0.146
+
+
+# TODO: MAYBE This file should be moved to a different package?????
+class AprilTagPoseEstimator:
```

Comment:
where do we get the reference images so that we can know the scale?

Reply:
We give the length QR code as an argument. That helps to compute scale